Tests/Packet++Test/Tests/SmtpTests.cpp

```diff
@@ -0,0 +1,240 @@
+#include "../TestDefinition.h"
+#include "../Utils/TestUtils.h"
+#include "EndianPortable.h"
+#include "EthLayer.h"
+#include "IPv4Layer.h"
+#include "IPv6Layer.h"
+#include "Packet.h"
+#include "SmtpLayer.h"
+#include "SystemUtils.h"
+#include "TcpLayer.h"
+
+PTF_TEST_CASE(SmtpParsingTests)
+{
+	timeval time;
+	gettimeofday(&time, nullptr);
+
+	// Command
+	READ_FILE_AND_CREATE_PACKET(1, "PacketExamples/smtpCommand.dat");
+	pcpp::Packet smtpPacket1(&rawPacket1);
+	pcpp::SmtpRequestLayer *smtpLayer1 = smtpPacket1.getLayerOfType<pcpp::SmtpRequestLayer>();
```

Comment:
nit: we can use `auto` instead of `pcpp::SmtpRequestLayer *` - here and in other places in this test